tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -128,6 +213,45 @@
         ],
         validity_error=EOFException.STACK_UNDERFLOW,
     ),
+    Container(
+        name="retf_stack_validation_2",
+        sections=[
+            Section.Code(code=Op.CALLF[1] + Op.STOP, max_stack_height=2),
+            Section.Code(
+                code=Op.PUSH0 * 3 + Op.RETF,
+                code_outputs=2,
+                max_stack_height=3,
+            ),
+        ],
+        expected_bytecode="ef000101000802000200040004040000000080000200020003e30001005f5f5fe4",
+        validity_error=EOFException.STACK_HIGHER_THAN_OUTPUTS,
+    ),
+    Container(
+        name="retf_variable_stack_2",
+        sections=[
+            Section.Code(code=Op.CALLF[1] + Op.STOP, max_stack_height=1),
+            Section.Code(
+                code=Op.PUSH0 + Op.PUSH1[0] + Op.RJUMPI[2] + Op.PUSH0 * 2 + Op.RETF,
```

Comment:
```suggestion
                code=Op.PUSH0 + Op.PUSH1[0] + Op.RJUMPI[1] + Op.PUSH0 + Op.RETF,
```
We should also include the case where it overflows by one stack item only.

Reply:
Also similar comment to one in the previous test function, we should add a variant where it overflows in the jumping branch, for example:
```python
Op.PUSH0 + Op.PUSH1[0] * 2 + Op.RJUMPI[1] + Op.POP + Op.RETF
```

Reply:
I added these two cases here: https://github.com/ethereum/execution-spec-tests/blob/3f30efd14aaf79e252b23168b3b2317031c3a9a5/tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py#L247-L270